test/python/algorithms/time_evolvers/variational/test_var_qite.py

```diff
@@ -79,14 +79,14 @@ def test_run_d_1_with_aux_ops(self):
         ]
 
         thetas_expected_shots = [
```

Comment:
This is a horrible test, we really need to fix this... but why did these change in this PR?

Reply:
Yes... I don't think we need to test `thetas`. I made changes to it because it returned different values for the shot-based backend than before. This is because the circuit order to be executed in `LinCombQGT` has changed, resulting in a change in the results with the random seed. (Previously, it iterated on set for parameters to be differentiated, but now it iterates on list.)